### PR TITLE
zcs-11694 :Add files shared with me system folder script

### PR DESCRIPTION
**Pull request by gabhinavTI**
Add files shared with me system folder script

### CODE REVIEW

**Review by dasiyogesh — APPROVED** [review]

File: src/db/migration/migrate20220729-FilesShareWithMeFolder.pl [hunk 1]
```diff
@@ -0,0 +1,155 @@
+
```
Comment (silentsakky) [inline src/db/migration/migrate20220729-FilesShareWithMeFolder.pl:1]: remove this line

File: src/db/migration/migrate20220729-FilesShareWithMeFolder.pl [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#!/usr/bin/perl
+
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2021 Synacor, Inc.
```
Comment (silentsakky) [inline src/db/migration/migrate20220729-FilesShareWithMeFolder.pl:7]: 2022

File: src/db/migration/migrate20220729-FilesShareWithMeFolder.pl (hunk 1 above)
Comment (gabhinavTI) [reply src/db/migration/migrate20220729-FilesShareWithMeFolder.pl]: done

File: src/db/migration/migrate20220729-FilesShareWithMeFolder.pl (hunk 2 above)
Comment (gabhinavTI) [reply src/db/migration/migrate20220729-FilesShareWithMeFolder.pl]: updated

**Review by silentsakky — APPROVED** [review]

**Pull request merged**